File: app/src/main/java/com/theappsolutions/boilerplate/ui/settings/SettingsView.java
package com.theappsolutions.boilerplate.ui.settings;

import com.theappsolutions.boilerplate.ui.base.BaseView;

/**
 * @author Severyn Parkhomenko s.parkhomenko@theappsolutions.com
 * @copyright (c) 2018 TheAppSolutions. (https://theappsolutions.com)
 */
public interface SettingsView extends BaseView {

    void startAuthClearTop();

    void fillView(boolean autologinEnabled);
}

File: app/src/main/java/com/theappsolutions/boilerplate/ui/settings/SettingsView.kt
package com.theappsolutions.boilerplate.ui.settings

import com.theappsolutions.boilerplate.ui.base.BaseView

/**
 * @author Severyn Parkhomenko s.parkhomenko@theappsolutions.com
 * @copyright (c) 2018 TheAppSolutions. (https://theappsolutions.com)
 */
interface SettingsView : BaseView {

    fun startAuthClearTop()

    fun fillView(autologinEnabled: Boolean)
}
